usage/jsgui/src/main/webapp/assets/css/brooklyn.css

```diff
@@ -25,7 +25,6 @@ BODY {
 textarea {
     white-space: pre;
     word-wrap: normal;
-    overflow-x: scroll;
```

Comment:
i think we rely on textareas scrolling e.g. when we show a blueprint or other formatted code in a textarea.

it perhaps would be better to have a special style for this but if we're removing this style have you made sure that no textareas suffer bad consequences?  if needed either update them or if you need a different `overflow-x` behaviour in the codemirror views define a new css rule.